proto/ignition/msgs/light.proto

```diff
@@ -61,4 +61,7 @@ message Light
 
   /// \brief Unique id of light's parent
   uint32 parent_id              = 17;
+
+  /// \brief light intensity
+  float intensity               = 18;
```

Comment:
can we default this to `1.0`?

Reply:
Explicit default values are not allowed in proto3.

Reply:
@iche033 can we merge this then?

Reply:
maybe we can force this as a required field

Reply:
ok I see, lets leave it as is then.